x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/index/IndexResolver.java

```diff
@@ -795,4 +800,24 @@ private GetAliasesRequest createGetAliasesRequest(FieldCapabilitiesResponse resp
         // everything checks
         return emptyMap();
     }
+
+    private class RemoteClusterResolver extends RemoteClusterAware {
```

Comment:
Can be made static as well.

Reply:
I've removed the class entirely, as the calling code has been removed. (It'll be re-added as static class for the SQL CCS.)